src/py/flwr/cli/new/templates/flower.toml.tpl

```diff
@@ -0,0 +1,9 @@
+[flower]
+name = "$project_name"
+version = "1.0.0"
+description = ""
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.ai>"]
+
+[components]
+app = "$project_name.main:app"
```

Comment:
We need both `serverapp` and `clientapp` here. Paths depend on whether we want everything in `main` or have it separated into `server.py` and `client.py`:
```suggestion
serverapp = "$project_name.server:app"
clientapp = "$project_name.client:app"
```